cores/esp32/esp32-hal-ledc.c

```diff
@@ -226,3 +226,7 @@ void analogWrite(uint8_t pin, int value) {
     ledcWrite(pin_to_channel[pin] - 1, value);
   }
 }
+
+int8_t analogGetChannel(uint8_t pin) {
+    return pin_to_channel[pin] - 1;
+}
```

Comment:
Can you fix the missing new line at the end of the file please?

Reply:
Please check if its ok now :)

Reply:
Looks good now :)